src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -0,0 +1,335 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_load.data_plane.utils import completers, models, utils, validators
+from azure.cli.core.commands.parameters import (
+    get_generic_completion_list,
+    get_resource_name_completion_list,
+    quotes,
+    resource_group_name_type,
+)
+from knack.arguments import CLIArgumentType
+
+quote_text = f"Use {quotes} to clear existing {{}}."
+
+# Common arguments
+resource_group = resource_group_name_type
+
+load_test_resource = CLIArgumentType(
+    options_list=["--load-test-resource", "--name", "-n"],
+    type=str,
+    required=True,
+    completer=get_resource_name_completion_list("Microsoft.LoadTestService/LoadTests"),
+    help="Name or ARM resource ID of the load test resource.",
+)
+
+wait = CLIArgumentType(
+    options_list=["--wait"],
+    action="store_true",
+    default=False,
+    help="Wait for the Long-Running Operation to complete.",
+)
```

Comment:
We use `--no-wait` instead.

Reply:
There are few reasons to move to wait instead of no-wait.

1. no-wait's initial response is not getting returned to output CMD.
2. Our portal behavior is similar to wait. We won't wait in create command for file upload.
3. For test run command generally, it can be of any duration (in hours too), so if user wants to wait they should provide wait explicitly otherwise they might get stuck for hours unwillingly.

Reply:
Updated as suggested